src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -4357,6 +4357,19 @@ components:
               type: array
               items:
                 $ref: "#/components/schemas/Address"
+            balance_change_resources_filter:
```

Comment:
Can we also tweak a few other places:
* Could we add a description of these filters to the `Supported filters` section of the examples?
* Could we update the `manifest_resources_filter` to recommend that integrators likely wish to use `balance_change_resources_filter` instead in most cases, as it also captures resource changes which aren't in the manifest.